lbz/lambdas/enums.py

```diff
@@ -0,0 +1,33 @@
+from typing import Iterable
+
+
+class LambdaSource:
+    API_GW = "api_gw"
+    DIRECT = "direct_lambda_request"
+    DYNAMODB = "dynamodb"
+    EVENT_BRIDGE = "event_bridge"
+    S3 = "s3"
+    SQS = "sqs"
+
+    @classmethod
+    def standard_aws_sources(cls) -> Iterable[str]:
+        return cls.DYNAMODB, cls.S3, cls.SQS
+
+    @classmethod
+    def get_source(cls, event: dict) -> str:
+        # FYI: AWS is very inconsistent in its way to provide the source information
+        if event.get("httpMethod") is not None:
+            return cls.API_GW
+        if event.get("invoke_type") == cls.DIRECT:
+            return cls.DIRECT
+        if event.get("detail-type") is not None:
```

Comment:
[Previously](https://github.com/pdyba/lambdalizator/pull/64#discussion_r966008901), I asked you to put here TODO to verify whether `detail-type` is a standard 😉 

Unless you have the answer already.